framework/src/userobject/LayeredBase.C

```diff
@@ -101,7 +106,10 @@ LayeredBase::LayeredBase(const InputParameters & parameters)
   if (!_interval_based && _sample_type == 1)
     mooseError("'sample_type = interpolate' not supported with 'bounds' in ", _layered_base_name);
 
-  if (_layered_base_params.isParamValid("block"))
+  if (_layered_base_params.isParamValid("layer_bounding_block"))
+    _blocks = _layered_base_subproblem.mesh().getSubdomainIDs(
```

Comment:
Does it make sense to rename `_blocks` to something like `_layer_bounding_blocks` to make it clear what it is being used for in this context?

Reply:
I am okay with this change.